<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>134</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>142</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GFWL-stripper</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>nogfwl.ico</normaloff>nogfwl.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="game_selection"/>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="filename_lineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browse_button">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="start_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>browse_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>processFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>game_selection</sender>
   <signal>activated(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>selectGame(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectFolder()</slot>
  <slot>processFolder()</slot>
  <slot>selectGame(int)</slot>
 </slots>
</ui>
